charts/clamav/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "1.8"
 description: An Open-Source antivirus engine for detecting trojans, viruses, malware & other malicious threats.
 name: clamav
-version: 1.0.11
+version: 1.0.12
```

Comment:
Could you bump minor version instead of patch version? This is a new feature.

Reply:
updated version to 1.1